tests/utils.py

```diff
@@ -14,10 +14,10 @@
 from ddtrace import Tracer
 from ddtrace.constants import SPAN_MEASURED_KEY
 from ddtrace.ext import http
-from ddtrace.internal._encoding import MsgpackEncoder
 from ddtrace.internal.compat import httplib
 from ddtrace.internal.compat import parse
 from ddtrace.internal.compat import to_unicode
+from ddtrace.internal.encoding import Encoder
```

Comment:
> Is this change required for this PR? If not I'd keep the explicit import of `MsgpackEncoder` to avoid breaking tests if the default value of the `Encoder` variable should ever change. We'll then sort this out in the v0.5 PR.

Same here